test/conftest.py

```diff
@@ -87,15 +87,6 @@ def pytest_addoption(parser):
     )
 
 
-@pytest.fixture(scope="session")
-def pudl_env(pudl_input_output_dirs):
```

Comment:
Looks like there are several remaining references to this fixture in `conftest.py`

Reply:
Thanks, dropped those.